lib/web/terminal.go

```diff
@@ -1032,6 +946,18 @@ type WSStream struct {
 	// decoder is used to decode UTF-8 strings.
 	decoder *encoding.Decoder
 
+	// resizeC a channel to forward resize events so that
+	// they happen out of band and don't block reads
+	resizeC chan Envelope
+	// fileTransferRequestC is a channel to facilitate requesting a file transfer
+	fileTransferRequestC chan Envelope
+	// fileTransferDecisionC is a channel to facilitate responding to a file transfer
+	// with an approval or denial
+	fileTransferDecisionC chan Envelope
```

Comment:
Can we not keep these fields here? WSStream is used by Terminal and Command Handlers. The second one doesn't need them. IMO we should try to find an abstraction that holds only the required fields in a struct.